.github/workflows/cd.yml

```diff
@@ -46,13 +48,29 @@ jobs:
         run: |
           aws ecs describe-task-definition --task-definition ${{ vars.ECS_TASK_DEFINITION }} --query taskDefinition > ${{ vars.ECS_TASK_DEFINITION_PATH }}
 
+      - name: Download Parameter Store Values
+        id: ssm-download
+        run: |
+          PARAMETERS_JSON=$(aws ssm describe-parameters --query "Parameters[?contains(Name, 'backend')].{Name:Name,ARN:ARN}" --output json | jq -c '.')
```

Comment:
Shoulds we use `contains` or maybe only check if it starts with that key? Just thinking that maybe some other keys are named backend and we end up pulling them as well

Reply:
I use `contains` because the name convention that Devops use is `project_name/(backend|frontend)/var_name` and I think is a good way to search, which scenery do you think could be retrieve keys that I don't want